src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -207,8 +215,22 @@ const NavHeader = (props) => {
               ]}>
               <TopicDomainHeader {...props} />
             </Animated.View>
+            <View style={styles.containerAction}>
+              <Animated.View style={{opacity: opacityHeaderAnimation}}>
+                {isFollow ? (
+                  <ButtonFollowing handleSetUnFollow={handleFollowTopic} />
+                ) : (
+                  <ButtonFollow handleSetFollow={handleFollowTopic} />
+                )}
+              </Animated.View>
+            </View>
           </>
-        )}
+        ) : null}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering logic has been changed here. Make sure that the new conditions correctly represent the intended behavior for when to show the follow button and when to show the unfollow button.



<!-- This is an auto-generated comment by OSS CodeRabbit -->